File: Java/duracao.java
import java.util.Scanner;

public class duracao {

	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);

		int duracao, horas, resto, minutos, segundos;

		System.out.print("Digite a duraçao em segundos: ");
		duracao = sc.nextInt();

		horas = duracao / 3600;
		resto = duracao % 3600;

		minutos = resto / 60;
		segundos = resto % 60;

		System.out.println(horas + ":" + minutos + ":" + segundos);
		sc.close();
	}
}

File: Python/duracao.py
tot_segundos =  int(input("Digite a duracao em segundos: "))

horas = tot_segundos // 3600
resto_horas = tot_segundos % 3600

minutos = resto_horas // 60
segundos = resto_horas % 60

print(f"{horas}:{minutos}:{segundos}")